debian/control

```diff
@@ -21,7 +21,6 @@ Architecture: any
 Pre-Depends: dpkg (>= 1.15.6)
 Depends: ${shlibs:Depends},
         ${misc:Depends},
-        chromium-browser,
         flatpak,
         gir1.2-flatpak-1.0,
```

Comment:
I think these two could also go since 85c62bcea560fa7dbec657cba950a84d88447ac6 removed the code that used them from this script?

Reply:
Updated, also removed `gir-glib` and `python3-gi` and added `libglib2.0-bin` (for the "gio" binary).